### PR TITLE
Added unit tests 

### PR DESCRIPTION
**Pull request by ayahusseini**
- unit tests added
- full coverage of dataframe_functions.py

### CODE REVIEW

File: dashboard/test_dataframe_functions.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    ("5h"),
+    ("12h"),
+    ("0h"),
+])
+def test_is_valid_time_interval_valid(interval):
+    """No exception should be raised for valid intervals"""
+    assert is_valid_time_interval(interval) is None
+
+
+@pytest.mark.parametrize("interval", [
```
Comment (megansam5) [inline dashboard/test_dataframe_functions.py:28]: Nice parametrize tests

**Review by megansam5 — APPROVED** [review]
Nice moving andtesting of functions!

File: dashboard/test_dataframe_functions.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
+"""Write unit tests for dataframe functions"""
```
Comment (YakMan101) [inline dashboard/test_dataframe_functions.py:1]: Extra line after docstring please

File: dashboard/dataframe_functions.py [hunk 3]
```diff
@@ -6,14 +6,28 @@
 AGGREGATES = ["mean", "count"]
 
 
+def is_valid_time_interval(interval: str) -> None:
```
Comment (YakMan101) [inline dashboard/dataframe_functions.py:9]: add a new line after docstring at the top of the document

**Pull request merged**